sdk/communication/azure-communication-administration/azure/communication/administration/_phone_number_administration_client.py

```diff
@@ -4,27 +4,29 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 # ------------------------------------
+from azure.communication.administration._phonenumber._generated.models import ReleaseStatus
 from azure.core.tracing.decorator import distributed_trace
 from azure.core.paging import ItemPaged
+from azure.core.polling import LROPoller
+from ._polling import ReleasePhoneNumberPolling, ReservePhoneNumberPolling, PurchaseReservationPolling
 
 from ._phonenumber._generated._phone_number_administration_service\
     import PhoneNumberAdministrationService as PhoneNumberAdministrationClientGen
 
 from ._phonenumber._generated.models import (
     AcquiredPhoneNumbers,
     AreaCodes,
-    CreateSearchResponse,
     LocationOptionsResponse,
     NumberConfigurationResponse,
     NumberUpdateCapabilities,
     PhoneNumberCountries,
     PhoneNumberEntities,
     PhoneNumberRelease,
-    PhoneNumberSearch,
+    PhoneNumberReservation,
     PhonePlanGroups,
     PhonePlansResponse,
     PstnConfiguration,
-    ReleaseResponse,
+    SearchStatus,
```

Comment:
Is this the status of the reservation? if yes, should we make it `ReservationStatus`?

Reply:
We are not exposing the `SearchStatus`. The status() from the LROPoller is returning a string instead of an enum. I will remove the exposure of  `SearchStatus` from __init__.py.

It is a little tricky to rename the enum using yaml. We will wait for swagger change to change the `SearchStatus` to `ReservationStatus` before GA.